<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QV_Objekt</class>
 <widget class="QWidget" name="QV_Objekt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QV_Objekt</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>961</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab_object">
    <attribute name="title">
     <string>Objekt</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Objekt ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Straße:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>PLZ:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Ort:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Etage:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Baujahr:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Größe:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Zimmer:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_object_id">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_object_name">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_object_street">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_object_zip">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_object_city">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_object_etage">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_object_year">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_object_size">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_object_zimmer">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>220</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_equipment">
    <attribute name="title">
     <string>Ausstattung</string>
    </attribute>
    <widget class="QTableWidget" name="tbl_equipment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>931</width>
       <height>192</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Objekt ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Erstellungsdatum</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fußboden</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sicherung</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Absicherung [A]</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bad|Dusche</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Waschmaschinenanschl.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Balkon vorhanden</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Kabelanschluss</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FTTH</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Heizungsart</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>weiteres</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>230</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Objekt ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Erstellungsdatum:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>250</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fußbodenart:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sicherung:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>280</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Absicherung in Ampere:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bad | Dusche</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>310</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Waschmaschinenanschluss:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>340</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Balkon:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>340</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kabelanschluss:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>370</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>FTTH:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>370</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Heizungsart:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>400</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Weiteres:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_e_id">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_e_objectid">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>230</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_e_creation">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_e_fussboden">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>250</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_e_fusevalue">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>280</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_e_waschmaschine">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>310</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_e_bad">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>310</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_e_balkon">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>340</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_e_kabelanschluss">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>340</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_ftth">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>370</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_heizung">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>370</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="tb_weiteres">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>400</y>
       <width>431</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_sicherung">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>280</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Schmelzsicherung</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sicherungsautomat</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btn_reset">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>510</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Zurücksetzen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_save">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>510</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Übernehmen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>510</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hinzufügen</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_tennant">
    <attribute name="title">
     <string>Mieter</string>
    </attribute>
    <widget class="QTableWidget" name="tbl_tennant">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>911</width>
       <height>192</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Objekt ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Adress ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vorname</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nachname</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Geburtstag</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Einzugsdatum</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Auszugsdatum</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>220</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Objekt ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>220</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Adress ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Vorname:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>250</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nachname:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>250</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Geburtstag:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Einzug:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>290</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Auszug:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_t_id">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>220</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_ob_id">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>220</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_t_aid">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>220</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_t_vorname">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_t_nachname">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>250</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_t_geburtstag">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>250</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_t_einzug">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>290</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_t_auszug">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>290</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_t_reset">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>360</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Zurücksetzen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_t_save">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>360</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Übernehmen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_t_add">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>360</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hinzufügen</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_accept">
   <property name="geometry">
    <rect>
     <x>864</x>
     <y>630</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Alles Übernehmen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>630</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Verwerfen</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
